### ISSUE TITLE
Enhancement: Embed form in non-portal area of Website, Ecommerce, etc.

### ISSUE THREAD

**Issue by walkerblackwell** [novacode-nl/odoo-formio#33]
A few things I've noticed that are barriers for implementation IRL.

1. Portal form submission area is buried inside of >My Account>Forms>Submit A Form and each submission (aka Form) is created as a unique url but only after going through the user portal. Currently there is no way to post a form on a single URL and have anyone go to that url and submit a form with recaptcha control similar to the way 99.999 percent of all forms work online today. Embedding the form in non-portal area is a must to me because how do people know where to go who are just coming to the website as public user without an account at all?

2. Form submissions are called "forms" in the backend. This should be changed to "Submissions" or "Form Entries" so they are not confused with the Builder. 

3. Is there plans in works to allow for forms to be added to the product on the ecommerce front-end to allow for form data entry before "add to cart"? This would make Odoo incredibly powerful.

best regards,
Walker

**Comment by bobslee** [comment]
Hi Walker,
Thanks for sharing your ideas. Let's elaborate more in this.

1. Indeed a good point. A (very) while ago such feature also came to my mind - the need for an Odoo website snippet. However the implementation details differ from the backend way. So I decided to put that on hold, unitl a need a PR or some funding arises.
If you could contribute e.g. by a new module, a patch (PR) or some way, that would be great!

2. I understand and could agree. However during my experience with "form projects" and applications/services, most users (internal, employees) appreciate to keep this term short, just "forms".
In case you would like to rename it, either:
- Extend the module (inherit) and override the terms
- Add translation (PO file)

3. Would be awesome indeed. But like I mentoined in (1), at moment I don't have any plans to implement yet.
Amongst my projects and customers, the Form.io stack is especially used in the backend environment i.e. orginisational processes. Example: Sales/Project quote analysis, CRM details, Employee surveys, Construction measuring, all kinds of checklists etc.

Greets,
Bob
